docs/documentation.md

```diff
@@ -0,0 +1,16 @@
+# Documentation 
+
+It follows a list of all the implemented workflows:
+
+- [`publish-next-to-ghpages`](../.github/workflows/publish-next-on-ghpages.yml):
+
+    publish Next.js website to GitHub Pages, this workflow has no inputs.
+
+- [`deploy-sphinx-docs-to-ghpages`](../.github/workflows/deploy-sphinx-docs-to-ghpages.yml):
+
+    deploy Sphinx documentation and publish it using GitHub Pages 
+    
+    **Inputs:** 
+
+    * **python-version** (*string*): (Mandatory) the Python version to be installed 
+    * **path-doc** (*string*): (Optional) documentation's path. The default value is `./doc`. 
```

Comment:
Move inputs description to dedicated files, as requested above.

Copy input description and examples layout from well-known actions: https://github.com/actions/checkout or https://github.com/actions/setup-python and similar.

Workflows are not invoked in the exact same way of actions, so account for differences, but being similar is legit to draw inspiration.